src/components/Composer.vue

```diff
@@ -335,7 +335,23 @@ export default {
 	},
 	computed: {
 		aliases() {
-			return this.$store.getters.accounts.filter((a) => !a.isUnified)
+			let cnt = 0
+			const aliases = []
+			const accounts = this.$store.getters.accounts.filter((a) => !a.isUnified)
+			accounts.forEach((account) => {
+				account.aliasId = null
```

Comment:
:warning: this is :fish:y. You should not mutate any data from the store. but you assign new props here. Vue will definitely warn you about this in the dev build. Any change of the state should go through an action and a mutation.

perhaps what you want to do is `.map(...)` the accounts to a new structure, as in, create new object